### DISCUSSION TITLE
UAT environment Set Up

### DISCUSSION THREAD

**Discussion by RahulSatanure** [ONDC-Official/ONDC-FIS-Specifications#21] [Q&A]
ONDC Team, 

Can we get IP's and Port Details of  ONDC's UAT environment for whitelisting purpose at our end.

Is ONDC is also required our UAT environments IP's for whitelisting purpose.

Also please let us know if any prerequisites are there for UAT environment set up.

**Comment by 92shreyansh** [comment]
There is no UAT. ONDC as a Network construct has has 2 lower enviornments stagging and pre-prod for NPs to interact with each other for non-production transaction testing.

And there is no specific IPs or whitelisting required.

I would really recommend you to attend the ONDC tech briefing call and understand the overall construct of ONDC, and would request to have this discusion used for specification related issue only.
